FlowCrypt/src/androidTest/java/com/flowcrypt/email/util/AccountDaoManager.kt

```diff
@@ -40,6 +41,11 @@ class AccountDaoManager {
       return getUserFromBaseSettings("no_backups@flowcrypt.test")
     }
 
+    fun getUserWithOrgRules(orgRules: OrgRules): AccountEntity {
```

Comment:
I suggest to rename this into `getTestUserWithOrgRules`.

Reply:
I'm not sure. This class is located in the tests package. So all things there relate to testing. And adding `Test` just will increase the method name.

If you don't have any new questions or recommendations please approve the current PR. It is blocking me a little to complete another one.

Reply:
ok, have not noticed it is in the test package